README.md

```diff
@@ -69,82 +94,155 @@ nb.scraps
 ```
 
 **scrapbook** will imply the storage format by the value type of any registered
-data translators. Alternatively, the implied storage format can be overwritten by
-setting the `storage` argument to the registered name (e.g. `"json"`) of a
-particular translator.
+data encoders. Alternatively, the implied encoding format can be overwritten by
+setting the `encoder` argument to the registered name (e.g. `"json"`) of a
+particular encoder.
 
 This data is persisted by generating a display output with a special media type
-identifying the content storage format and data. These outputs are not visible in
-notebook rendering but still exist in the document. Scrapbook then can rehydrate
-the data associated with the notebook in the future by reading these cell outputs.
+identifying the content encoding format and data. These outputs are not always
+visible in notebook rendering but still exist in the document. Scrapbook can
+then rehydrate the data associated with the notebook in the future by reading
+these cell outputs.
 
-### `sketch` to save _display output_
+#### With _display output_
 
-Display a named snap (visible display output) in a retrievable manner.
+To display a named scrap with visible display outputs, you need to indicate that
+the scrap is directly renderable.
 
-Unlike `glue`, `sketch` is intended to generate a visible display output
-for notebook interfaces to render.
+This can be done by toggling the `display` argument.
 
 ```python
-# record an image highlight
-sb.sketch("sharable_png", IPython.display.Image(filename=get_fixture_path("sharable.png")))
-# record a UI message highlight
-sb.sketch("hello", "Hello World")
+# record a UI message along with the input string
+sb.glue("hello", "Hello World", display=True)
 ```
 
-Like scraps, these can be retrieved at a later time. Unlike scraps, highlights
-do not carry any actual underlying data, keeping just the display result of some
-object.
-
-```python
-nb = sb.read_notebook('notebook.ipynb')
-# Returns the dict of name -> snap pairs saved in `nb`
-nb.snaps
-```
+The call will save the data and the display attributes of the Scrap object,
+making it visible as well as encoding the original data. This leans on the
+`IPython.core.formatters.format_display_data` function to translate the data
+object into a display and metadata dict for the notebook kernel to parse.
 
-More usefully, you can copy snaps from earlier notebook executions to re-display
-the object in the current notebook.
+Another pattern that can be used is to specify that **only the display data**
+should be saved, and not the original object. This is achieved by setting
+the encoder to be `display`.
 
 ```python
-nb = sb.read_notebook('notebook.ipynb')
-nb.copy_highlight("sharable_png")
+# record an image without the original input object
+sb.glue("sharable_png",
+  IPython.display.Image(filename=get_fixture_path("sharable.png")),
```

Comment:
What is get_fixture_path here?

Reply:
oh opps, that was taken from the test -- shouldn't be there